packages/admin/admin/src/form/ChipSelect.tsx

```diff
@@ -0,0 +1,201 @@
+import { ChevronDown } from "@comet/admin-icons";
+import {
+    Chip as MuiChip,
+    ChipProps,
+    ComponentsOverrides,
+    InputBase as MuiInputBase,
+    InputBaseProps,
+    MenuItem,
+    Select,
+    SelectChangeEvent,
+} from "@mui/material";
+import { css, Theme, useThemeProps } from "@mui/material/styles";
+import * as React from "react";
+
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+
+export type ChipSelectClassKey = "root" | "inputBase" | "inputRoot" | "chip" | "select";
+
+type OwnerState = {
+    fullWidth: boolean;
+};
+
+const Root = createComponentSlot("div")<ChipSelectClassKey, OwnerState>({
+    componentName: "ChipSelect",
+    slotName: "root",
+})(
```

Comment:
Please also add the `fullWidth` state using the `classesResolver`. 
Like the `highlighted` state in the docs example: https://next--comet-dxp-docs.netlify.app/docs/comet-core-development/creating-customizable-admin-components/#a-slots-classesresolver